gds/CMakeLists.txt

```
@@ -110,7 +80,7 @@ PUBLIC
 target_include_directories(cufile_stub
     PUBLIC
         $<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/include>
-        $<BUILD_INTERFACE:${GDS_INCLUDE_PATH}>
+        CUDAToolkit_INCLUDE_DIRS
```

I think we need to expand the value here
```suggestion
        ${CUDAToolkit_INCLUDE_DIRS}
```

---

let me go ahead and commit this and see if it resolves the existing build failure for wheels...